Server/mods/deathmatch/logic/CGame.cpp

```diff
@@ -2441,14 +2427,14 @@ void CGame::RelayNearbyPacket(CPacket& Packet)
                 if (!nearInfo.bInPureSyncSimSendList)
                 {
                     // If not in sim send list, do send here
-                    sendList.push_back(pSendPlayer);
+                    sendList.Insert(pSendPlayer);
                 }
             }
         }
     }
 
     // Relay packet
-    if (!sendList.empty())
+    if (!sendList.Empty())
```

Comment:
Wouldn't it make more sense to add that check to `CPlayerManager::Broadcast`?

Reply:
Well, there are very few scenarios where the sendList might end up being empty. Even without this `if` the code would probably run just as fast (or faster).